### ISSUE TITLE
Criar história do jogo

### ISSUE THREAD

**Issue by darkSasori** [cassiosantos/DungeonAndDevs#1]
Criar um historia para o jogo comecar a ser desenvolvido

**Comment by cassiosantos** [comment]
Pelo que entendi é um RPG mediaval. Seria legal manter ele bem simples, com poucos recursos mas que já fossem jogáveis.

**Comment by darkSasori** [comment]
Concordo, mas acho interessante ter uma historia de onde o jogador comeca, ter uns NPCs para gerar missoes, onde por exemplo ele tem que matar um dragao, mas para matar ele precisa melhorar os equipamentos, ai ele fala com outro NPCs para ganhar nivel para conseguir matar o dragao.

**Comment by cassiosantos** [comment]
Massa! Acho que o jogo poderia começar em uma vila inspirada nas vilas islandesas de 1800, algo como uma paisagem relacionada a um panorama vulcânico e gelado. O jogar seria um morador deste lugar que se depara com uma situação adversa.
